File: dayOne/JavaScript/LevelOne/PerfectSquaredNumbers.js
function getPerfectSquares(numbers) {
    const perfectSquares = [];
        
    for (const number of numbers) {
        if (number >= 0) {
            const root = Math.floor(Math.sqrt(number));
            
            if (root * root === number) {
                perfectSquares.push(number);
            }
        }
            
    }
    
    return perfectSquares;
}

const array = [4, 7, 9, 10, 16, 18];
const newArray = getPerfectSquares(array);

console.log(newArray);

File: dayOne/Python/LevelOne/perfect_squared_numbers.py
import math

def get_perfect_squared_numbers(list):

    perfect_squares = []
    
    for number in list:
        if number >= 0:
            root = int(math.sqrt(number))
            
            if root * root == number:
                perfect_squares.append(number)
                
    return perfect_squares
    
numbers = [4, 7, 9, 10, 16, 18];
new_list = get_perfect_squared_numbers(numbers)

print(new_list)
